templates/certified/devices.html

```diff
@@ -0,0 +1,189 @@
+{% extends "certified/base_certification.html" %}
+
+{% block title %}Certified hardware{% endblock %}
+
+{% block meta_description %}Devices that have been certified for use with Ubuntu.{% endblock meta_description %}
```

Comment:
Try and find some wording that defines the purpose of the content.

Reply:
I would rather leave this to the copy docs / PM, whatever I come up with will most likely be replaced soon.